internal/gengapic/custom_operation.go

```diff
@@ -130,6 +130,28 @@ func (g *generator) customOperationType() error {
 	p("}")
 	p("")
 
+	// Wait
+	p("// Wait blocks until the operation is complete, polling regularly")
+	p("// after an intial period of backing off between attempts.")
+	p("func (o *%s) Wait(ctx context.Context, opts ...gax.CallOption) error {", opName)
+	p("	 bo := gax.Backoff{")
+	p("    Initial: 1 * time.Second,")
+	p("    Max:     time.Minute,")
+	p("	 }")
+	p("	 for {")
```

Comment:
nit: indentation within the quotes is wonky (I know it doesn't affect the final product, but for readability here)

Reply:
Ah thanks for catching this, copy-paste resulted in some tab-characters sneaking in. Fixed.